go-controller/pkg/libovsdbops/transact.go

```diff
@@ -50,6 +50,10 @@ func TransactAndCheck(c client.Client, ops []ovsdb.Operation) ([]ovsdb.Operation
 
 	opErrors, err := ovsdb.CheckOperationResults(results, ops)
 	if err != nil {
+		// If there is only one error, keep it 'as is', inside a message wrap
+		if len(opErrors) == 1 {
```

Comment:
@kyrtapz @trozet help needed in case either of you have any golang tricks up your sleeve on how to express the `%w` when there are an array of errors.

We tried to convert the list given by CheckOperationResults into a wrapped error that can then be unwrapped on the other side to perform the error comparison and take action based on that, but didn't succeed. Ideally we'd like to get rid of this len hack and just go with returning the whole list somehow that can be un-parsed on the other side.

Reply:
If we really need to, we would inspect the errors returned by `CheckOperationResults` and map that to our own error types that express what that error means to us.

Reply:
ack. Since we are not looking at that error in this PR, we can defer this logic implementation for another time.